client/src/components/modal-forms/PhaseDialog.js

```diff
@@ -63,7 +63,12 @@ export const PhaseDialog = ({ onSubmit, onClose, open, content, isNew }) => {
         {({ submitForm }) => (
           <FormikForm>
             <Box>
-              <Field name='phaseName' component={RenderTextField} label='* Phase name' />
+              <Field
+                name='phaseName'
+                component={RenderTextField}
+                label='* Phase name'
+                placeholder='Type in here'
```

Comment:
Nitpick: could be more specifically relevant to the field's purpose. Something like "Enter a phase name", perhaps?

Reply:
Good suggestion, updated!